package.json

```diff
@@ -19,7 +19,8 @@
     "babel-register": "^6.6.0",
     "bower": "^1.7.7",
     "gulp": "^3.9.1",
-    "unigulp": "github:andersinno/unigulp#086c3c4844ec4979ddf72a5db4d96fa28256c9cd"
+    "unigulp": "github:andersinno/unigulp#086c3c4844ec4979ddf72a5db4d96fa28256c9cd",
+    "chart.js": "^2.1.5"
```

Comment:
Looks like all other front-endy resources are installed with bower. Maybe chart.js should be moved there, since it a front-end library? On the other hand it looks like `bootstrap` and `font-awesome` are also available from npm. For `fontkit-lobster` I couldn't find a straightforward package to install it from `npm`. I guess if we could install everything from npm there wouldn't be any need to use bower?